src/backend/db/database_manager.py

```diff
@@ -259,3 +296,131 @@ def check_customer_guid_exists(self, customer_guid):
         finally:
             session.close()
 
+    def add_custom_field(self, customer_guid, field_name, field_type, required):
+        customer_db_name = self.get_customer_db(customer_guid)
+        session = DatabaseManager._session_factory()
+
+        # Mapping field_type to SQL types
+        sql_type_map = {
```

Comment:
You dont need this mapping table as the initial field type itself will give you the actual value

Reply:
updated sir